green.py

```diff
@@ -0,0 +1,30 @@
+# Based on an example script written by Tony DiCola (tony@tonydicola.com),
```

Comment:
Would be more flexible if we could just have one file that takes a color code as input and can set any color. That way we cut back from 3 script files to 1 and we allow for more color choices.

Reply:
@alexwaibel  What would you think of a script that would take an int representing seconds and something representing the desired color and then display the color for int seconds and turn the lights off?

Reply:
That sounds good to me.